pageserver/src/tenant/timeline/walreceiver/walreceiver_connection.rs

```diff
@@ -209,14 +207,7 @@ pub(super) async fn handle_walreceiver_connection(
             replication_message = physical_stream.next() => replication_message,
         }
     } {
-        let replication_message = match replication_message {
-            Ok(message) => message,
-            Err(replication_error) => {
-                let expected_error = ignore_expected_errors(replication_error)?;
-                info!("Replication stream finished: {expected_error}");
```

Comment:
Now this is:

```
2023-06-02T11:22:30.177200Z  INFO wal_connection_manager{tenant_id=ae7f518773eb506c8aa1460873883e24 timeline_id=354c3460165bf13bba3cee3f4aab8409}:connection{node_id=1}: walreceiver connection handling ended: db error: ERROR: ending streaming to Some("pageserver") at 0/40BCE08, receiver is caughtup and there is no computes: ERROR: ending streaming to Some("pageserver") at 0/40BCE08, receiver is caughtup and there is no computes
```

And I guess it triggers the allowed_errors because we do just "grep" `(WARN|ERROR)` and that postgres::Error must inject that `ERROR` string there.

I think I already PR'd a proper log file parsing in https://github.com/neondatabase/neon/pull/3798#discussion_r1133697114 but @hlinnaka did not want it. Does @hlinnaka want to decide how to proceed here again?

Reply:
Trying with the stricter regex in [7bce70d](https://github.com/neondatabase/neon/pull/4402/commits/7bce70d1e0afd865f9fc948af48b0185471995c1) to see how many errors are left.

Reply:
Aah, silly me, before these were formatted with display, not display alternative.

Reply:
I was not expecting this to be in allowed_errors. In the end, I've now dropped the stricter regex, and just fixed the changed string.

Resolving.